src/languages/properties.js

```diff
@@ -0,0 +1,22 @@
+/*
+Language: Property file
+Contributors: Raffaele Sgarro <raffaeleSgarro@gmail.com>
+Category: config
+Description: Property files used in Java. A simple key value pair separated by '='
+*/
+function(hljs) {
+  return {
+    contains: [
+      hljs.HASH_COMMENT_MODE,
+      {
+        className: 'variable',
+        begin: /^[^=]+/,
```

Comment:
Somehow the pull request broke the build and so won't be merged as-is. Can't find the failing test, though, so I'm not sure I want to keep on trying